docs/reST/ref/cursors.rst

```diff
@@ -138,6 +138,8 @@ The following strings can be converted into cursor bitmaps with
    | :sg:`Cursor(size, hotspot, xormasks, andmasks) -> Cursor`
    | :sg:`Cursor(hotspot, surface) -> Cursor`
    | :sg:`Cursor(constant) -> Cursor`
+   | :sg:`Cursor(Cursor) -> copies the Cursor object as a parameter`
```

Comment:
I still don't think this needs documentation

Reply:
But you did document a similar constructor for pygame.Color objects, why not do the same here ?

Reply:
Fine. But it should be `-> Cursor` just like everything else.